content/blog/grpc-cloud-run.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Not just for HTTP anymore: gRPC comes to Cloud Run"
+date: 2020-03-17
+author:
+  name: Wenlei He
+  link: https://github.com/wlhee
+---
+
+*This is a cross-post of [an article](https://cloud.google.com/blog/products/compute/serve-cloud-run-requests-with-grpc-not-just-http)
```

Comment:
If this is just a cross-post, why distract the reader. I'd suggest moving this to the end of the page, something like this:

> ...
> _Google Cloud blog [cross-post](https://cloud.google.com/blog/products/compute/serve-cloud-run-requests-with-grpc-not-just-http)._